tests/baselines/reference/outModuleConcatUmd.sourcemap.txt

```diff
@@ -16,11 +16,112 @@ sourceFile:tests/cases/compiler/ref/a.ts
 >>>// This should error

```

Comment:
This is another case where I'd rather we not accept this file as it uses `outFile`.